docs/types.md

```diff
@@ -283,3 +283,22 @@ Format (each line is an [extended value](#extended-value) which takes [`1..5`] b
 | channel `N` diff `2` |
 
 Here it's a structure for `N` channels with for example `2` hours diff values.
+
+
+## Channel values by days
+
+Similarly to [channel values](#channel-values) it's a sequence of [extended values](#extended-value) to store channel pulse counter's values for days.
+<br>
+Format (each line is an [extended value](#extended-value) which takes [`1..5`] bytes):
+
+| Data                      |
+| ------------------------- |
+| channel `1` day `1` value |
```

Comment:
sadly this approach is not welcomed anymore

section "## Channel values with diffs" will also disappear in the next pull request

please use this approach
https://github.com/jooby-dev/jooby-docs/blob/main/docs/commands/uplink/ExAbsDayMC.md#format

Reply:
fixed